openupgradelib/openupgrade_merge_records.py

```diff
@@ -368,7 +370,12 @@ def _adjust_merged_values_orm(env, model_name, record_ids, target_record_id,
         if not field.store or field.compute or field.related:
             continue  # don't do anything on these cases
         op = field_spec.get(field.name, False)
-        _list = all_records.mapped(field.name)
+        try:
```

Comment:
You should make the try then only for that case, but not in general, or you will be masking too much.